pkg/controllers/inventory_controller.go

```diff
@@ -19,6 +19,7 @@ package controllers
 import (
 	"context"
 	"fmt"
+	"reflect"
```

Comment:
this is auto generated by kubebuilder when defining the crd and controllers. This is used for generating the rbac manifests: https://book.kubebuilder.io/reference/markers/rbac.html

This can be safely dropped